### PR TITLE
adding prediction data and fixing training columns used

### PR DESCRIPTION
**Pull request by bradmiro**
## Description

Adds prediction data to data-science-onramp tutorial and makes a fix in the utils to train on the correct columns.

## Checklist
- [x] I have followed [Sample Guidelines from AUTHORING_GUIDE.MD](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/AUTHORING_GUIDE.md)
- [x] README is updated to include [all relevant information](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/AUTHORING_GUIDE.md#readme-file)
- [ ] **Tests** pass:   `nox -s py-3.6` (see [Test Environment Setup](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/AUTHORING_GUIDE.md#test-environment-setup))
- [ ] **Lint** pass:   `nox -s lint` (see [Test Environment Setup](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/AUTHORING_GUIDE.md#test-environment-setup))
- [ ] These samples need a new **API enabled** in testing projects to pass (let us know which ones)
- [ ] These samples need a new/updated **env vars** in testing projects set to pass (let us know which ones)
- [ ] Please **merge** this PR for me once it is approved.
- [ ] This sample adds a new sample directory, and I updated the [CODEOWNERS file](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/.github/CODEOWNERS) with the codeowners for this sample

### CODE REVIEW

**Review by leahecole — APPROVED** [review]

File: data-science-onramp/ai-platform/modules/trainer/utils.py [hunk 1]
```diff
@@ -34,8 +34,8 @@ def load_data(
 
     # [START ai_platform_utils_extract_ft]
     # Extract feature and target columns
-    feature_col = df.iloc[:, :-1]
-    target_col = df.iloc[:, -1:]
+    feature_col = df.iloc[:, :10]
+    target_col = df.iloc[:, 10:]
```
Comment (ivanmkc) [inline data-science-onramp/ai-platform/modules/trainer/utils.py:38]: Minor nit: I assume you'd want to only take the last column instead of 10 to the end?

This technically is the same if there are only 10 columns but could leave to inadvertent bugs if number of features change.

Why the change in any case?

**Review by ivanmkc — APPROVED** [review]
LGTM

File: data-science-onramp/ai-platform/modules/trainer/utils.py (hunk 1 above)
Comment (bradmiro) [reply data-science-onramp/ai-platform/modules/trainer/utils.py:38]: The target columns are sparse one-hot encoded vectors and only the first 10 columns are features. We provide these dimensions dynamically when creating the model.
